### DISCUSSION TITLE
Controllo basculante garage

### DISCUSSION THREAD

**Discussion by excalibur470** [indomus/forum#2526] [GENERALE - Discussioni generiche]
Una saluto a tutta la community.
Vorrei controllare la basculante del garage e ho provato ad acquistare uno di quei moduli Tuya, precisamente un PJ-ZGD01 e l'ho integrato in HA, via Zigbee2MQTT.
Il problema è che non ho capito come diavolo funziona questo "coso".
L'oggetto prevede un reed per rilevare la condizione di porta chiusa e un contatto pulito per azionare la porta.
In HA mi trovo un comando Trigger e lo stato porta.
Il problema è che, azionando Trigger, viene inviato un impulso per cui il motore parte ma, subito dopo ne arriva un altro, per cui il motore si ferma. Azionandolo di nuovo ovviamente la la stessa cosa nel verso opposto.
Qualcuno di voi ha qualche esperienza in merito?
Grazie

**Comment by excalibur470** [comment]
Sono proprio l'unico ad averci provato?

**Reply by nromagna** [reply]
Ciao Ex, allora se il tuo problema è avere ANCHE lo stato del garage aperto/chiuso non ti posso aiutare, ma per il semplice comando di impulso per apri/chiudi io ho risolto con un Sonoff a 12V (o 24 ora non ricordo) che semplicemente da il comando (con un Inching di 0.5sec).
Il mio garage si chiude in automatico dopo 3 minuti per cui non mi serve sapere lo stato e questa modifica di chiudersi un automatrico lo fa la centralina del garage (presumo di tutti i garage/cancelli). Ecco tutto qua.

PS  pro di avrere questa chiusura automatica che se anche ti dimentici dopo 3 minuti il portone si chiude (ovvio!) 
      contro che NON posso lasaciare il basculante aperto a metà (per far passare aria) ma è sempre aperto o sempre chiuso...
